Tiltfile

```diff
@@ -9,7 +9,7 @@ if str(local("command -v " + kubectl_cmd + " || true", quiet = True)) == "":
 # set defaults
 settings = {
     "debug": {
-        "enabled": False,
+        "enabled": True,
```

Comment:
You can use a local tilt setting file for this instead of modifying the actual tilt file. :)

Reply:
@Skarlso good catch, this is resolved.